addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -5251,50 +5282,105 @@ registry.layout_column = SnippetOptionWidget.extend({
      *
      * @private
      * @param {jQuery} $row - the row in which to update the columns
-     * @param {integer} count - positif to add, negative to remove
+     * @param {integer} nbColumns - the number of columns requested
      */
-    _updateColumnCount: async function ($row, count) {
-        if (!count) {
+    _updateColumnCount: async function ($row, nbColumns) {
```

Comment:
As you rewrite this, you can do it in the new way
`async _updateColumnCount(...) {`